loki/transformations/single_column/tests/test_scc_vector.py

```diff
@@ -0,0 +1,527 @@
+# (C) Copyright 2018- ECMWF.
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+# In applying this licence, ECMWF does not waive the privileges and immunities
+# granted to it by virtue of its status as an intergovernmental organisation
+# nor does it submit to any jurisdiction.
+
+import pytest
+
+from loki import Subroutine, Sourcefile, Dimension, fgen
+from loki.frontend import available_frontends
+from loki.ir import (
+    FindNodes, Assignment, CallStatement, Conditional, Comment, Loop,
+    Pragma, Section
+)
+from loki.transformations.single_column import (
+    SCCDevectorTransformation, SCCRevectorTransformation, SCCVectorPipeline
+)
+
+
+@pytest.fixture(scope='module', name='horizontal')
+def fixture_horizontal():
+    return Dimension(
+        name='horizontal', size='nlon', index='jl',
+        bounds=('start', 'end'), aliases=('nproma',)
+    )
+
+@pytest.fixture(scope='module', name='horizontal_bounds_aliases')
+def fixture_horizontal_bounds_aliases():
+    return Dimension(
+        name='horizontal_bounds_aliases', size='nlon', index='jl',
+        bounds=('start', 'end'), aliases=('nproma',),
+        bounds_aliases=('bnds%start', 'bnds%end')
+    )
```

Comment:
...and this fixture seems unused here :)

Reply:
I think this one is still used:
https://github.com/ecmwf-ifs/loki/blob/naml-scc-test-reshuffle/loki/transformations/single_column/tests/test_scc_vector.py#L131

Reply:
Sorry, flagged it on the wrong file 🤦 I think that fixture is unused in test_scc_hoist.py

Reply:
Ahhh, gotcha now! Just "fixed the fix" and updated for one more round of CI goodness. :sunglasses: